arangod/Replication2/ReplicatedLog/Components/LogFollower.cpp

```diff
@@ -131,14 +131,15 @@ FollowerManager::~FollowerManager() {
 
 auto FollowerManager::getStatus() const -> LogStatus {
   auto commitIndex = commit->getCommitIndex();
-  auto log = storage->getCommittedLog();
+  auto mapping = storage->getTermIndexMapping();
```

Comment:
This method returns `onDiskMapping`. As far as I can see from the code, it is the same as `spearheadMapping`, so it seems fine that it's used as the spearhead value below. I'm writing this comment as a double check.

Reply:
this is a bit misleading. The spearhead of a follower is the highest log index that is persisted on disk. However, in the storage manager we can have multiple storage transactions that depend on each other. The virtual result of the last transaction is also called spearhead. If one of the transactions fails, we abort them in a cascade.